server/api/serializers.py

```diff
@@ -2,52 +2,80 @@
 from rest_framework import serializers
 from .models import *
 
-# Serializers to support seralizing User and Group objects
+"""
+The HyperlinkedModelSerializer class is similar to the ModelSerializer
+class except that it uses hyperlinks to represent relationships, rather
+than primary keys. 
+"""
+
+"""
+Serializers to support seralizing User and Group objects
+"""
 class UserSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = User
-        fields = ['email', 'first_name', 'last_name', 'groups', 'is_staff']
+        fields = ['email', 'first_name', 'last_name', 'groups', 'is_staff', 'linked_requests']
+        
+        depth = 1 # To include details of linked_requests
 
 class GroupSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = Group
         fields = ['admin', 'user']
 
-# Serializers to support seralizing EquipmentType objects
+
+"""
+Serializers to support serializing EquipmentCategory objects
+"""
+class EquipmentCategorySerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = EquipmentCategory
+        fields = "__all__"
+
+
+"""
+Serializers to support seralizing EquipmentType objects
+"""
 class EquipmentItemSerializerSimple(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = EquipmentItem
         fields = ['id', 'url']
 
 class EquipmentTypeSerializer(serializers.HyperlinkedModelSerializer):
-    items = EquipmentItemSerializerSimple(many=True, required=False)
+    items = EquipmentItemSerializerSimple(many=True, required=False, read_only=True)
 
     class Meta:
         model = EquipmentType
-        fields = ['name', 'items', 'url']
+        fields = "__all__"
```

Comment:
Love shortcuts! In general, we would want to explicitly set what fields we want to expose due to security considerations since we may add a field later that should be private. However, for this project this is fine.